.github/workflows/platform_tests-command.yml

```diff
@@ -20,10 +20,6 @@ on:
         description: 'Get info of forked repository and branch'
         required: false
 
-permissions:
-  contents: read
-  pull-requests: write
-
```

Comment:
I want to merge this for testing purposes. If having this change committed to `master` does not fix the platform tests, this will mean that the problem is with the permission scope on our security tokens.